src/gam/__init__.py

```diff
@@ -6186,6 +6186,34 @@ def getVacation(users):
                     print('None')
 
 
+def doSignout(users):
```

Comment:
move these into gapi/directory/users.py where all the other user stuff will eventually land. Names should be signout() and turn_off_2sv().